docs/internals/protocol/messages.rst

```diff
@@ -182,6 +182,12 @@ Format:
 
 .. eql:struct:: edb.testbase.protocol.CommandComplete
 
+Known headers:
+
+* 0x1001 ``CAPABILITIES``: ``uint64`` -- capabilities actually used in the.
```

Comment:
Can't parse the dangling "the" in "capabilities actually used in the." Please fix.